modules/sdk-core/src/bitgo/inscription/iInscription.ts

```diff
@@ -0,0 +1,15 @@
+import { PreparedInscriptionRevealData, TapLeafScript } from '@bitgo/utxo-ord';
+
+export interface IInscription {
+  prepareReveal(inscriptionData: Buffer, contentType: string, feeRate: number): Promise<PreparedInscriptionRevealData>;
+
+  // same response as wallet.submitTransaction
+  signAndSend(
+    walletPassphrase: string,
+    tapLeafScript: TapLeafScript,
+    unsignedCommitTx: string,
+    commitTransactionUnspents: any[],
```

Comment:
this should have a stricter type than `any`

Reply:
I used a sample response from /build to see what unspent type should be. Not sure if that covers all the cases though

Reply:
since we didnt have an existing type. We have txInfo set as unknown atm.